case-repository/src/main/java/eu/itesla_project/cases/EntsoeAndXmlCaseRepository.java

```diff
@@ -37,7 +37,7 @@ public EntsoeAndXmlCaseRepository(EntsoeCaseRepositoryConfig config, Computation
         super(config,
                 Arrays.asList(new EntsoeFormat(Importers.getImporter("CIM1", computationManager), "CIM"),
                         new EntsoeFormat(Importers.getImporter("UCTE", computationManager), "UCT"), // official ENTSOE formats)
-                        new EntsoeFormat(Importers.getImporter("XML", computationManager), "XML")), // IIDM XML format
+                        new EntsoeFormat(Importers.getImporter("XML", computationManager), "IIDM")), // IIDM XML format
```

Comment:
You right, I think you can change the name of the importer to use "XIIDM"

About EntsoeAndXmlCaseRepository class name, I think i will do a new issue: Entsoe is not a format contrary to IIDM. I think this class could be split in two classes: the first one will be an abstract class which implement the recursive logic for scanning the repository (Format/BasecaseType/YYYY/MM/DD). A name proposal could be "RecursiveRepository" or anything else. A concrete subclass could be configured to use CIM/UCTE and XIIDM importers.
Another subclass could be fully customizable: for each country, one could select the importers to use with a default behaviour... Maybe these two subclasses could be the same class. To be continued :)